app/src/main/java/com/arygm/quickfix/ui/search/SearchWorkerResult.kt

```diff
@@ -135,84 +118,31 @@ fun SearchWorkerResult(
           ),
       )
 
-  // ==========================================================================//
-  // ============ TODO: REMOVE NO-DATA WHEN BACKEND IS IMPLEMENTED ============//
```

Comment:
As the backend hasn't been implemented yet, I think it's best to leave it in for the application to work a bit.

Reply:
The application is working as before, all those are defined under, at line `254` although they are a bit less populated than in the original file all the functionality is there and they are shown